examples/nidcpower_source_dc_voltage/measurement.py

```diff
@@ -126,15 +127,18 @@ def cancel_callback():
                 try:
                     channels.wait_for_event(nidcpower.enums.Event.SOURCE_COMPLETE, timeout=0.1)
                     break
-                except nidcpower.DriverError as e:
+                except nidcpower.Error as e:
```

Comment:
`DriverError` is defined here: https://github.com/ni/nimi-python/blob/master/generated/nidcpower/nidcpower/errors.py

but it's not aliased into the `nidcpower` module: https://github.com/ni/nimi-python/blob/master/generated/nidcpower/nidcpower/__init__.py

`nidcpower.Error` isn't guaranteed to have a `code` field, so change this to `nidcpower.errors.DriverError`

Reply:
The same is true for `nifgen`, so please update the error check you added yesterday to use `nifgen.errors.DriverError`

Reply:
Fixed. Tested the DCPower example again and `nidcpower.errors.DriverError` worked.